vendor/autoload.php

```diff
@@ -2,6 +2,6 @@
 
 // autoload.php @generated by Composer
 
-require_once __DIR__ . '/composer' . '/autoload_real.php';
+require_once __DIR__ . '/composer/autoload_real.php';
```

Comment:
interesting, it looks like changes may stem from apcu settings in php:
https://github.com/composer/composer/commit/6d4e60b9914e5514c49f4b19ee2b664c1d477605
(I think it's a nonissue though, since this works fine on my system)